examples/old-photos/.saturn/saturn.json

```
@@ -0,0 +1,24 @@
+{
+    "name": "example-restoring-photos",
+    "image_uri": "saturncloud/saturn:2021.11.10",
+    "description": "Restoring old photos using Saturn Cloud",
```

We should more explicitly credit the Microsoft libraries here, like

```suggestion
    "description": "Restoring old photos using a neural network from Microsoft",
```